SignallingWebServer/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # Use the current Long Term Support (LTS) version of Node.js
 FROM node:lts
 # Copy the signalling server and frontend source code from the build context
+COPY package.json package.json
```

Comment:
This was needed in the Dockerfile to make it use the new NPM workspaces way of doing things.